packages/ingest/test/test-HttpsProviderClient.js

```diff
@@ -212,3 +236,33 @@ test('HttpsProviderClient supports basic auth with redirects for sync', async (t
     await recursivelyDeleteS3Bucket(destinationBucket);
   }
 });
+
+test('HttpsProviderClient supports basic auth for sync with redirect to different host/port', async (t) => {
+  const httpsProviderClient = new HttpProviderClient({
+    protocol: 'https',
+    host: '127.0.0.1',
+    port: t.context.server.sslPort,
+    certificateUri: `s3://${t.context.configBucket}/certificate.pem`,
+    username: basicUsername,
+    password: basicPassword,
+  });
+
+  const destinationBucket = randomString();
+  const destinationKey = 'syncedFile.hdf';
+  try {
+    await s3().createBucket({ Bucket: destinationBucket }).promise();
+    await httpsProviderClient.sync({
+      fileRemotePath: protectedFile2,
+      destinationBucket,
+      destinationKey,
+    });
+    t.truthy(fileExists(destinationBucket, destinationKey));
```

Comment:
Isn't `fileExists` async? It seems like it wasn't `awaited` before but... shouldn't it be?

Reply:
You are probably right. And the tests are only passing because they're doing `truthy` and an unresolved Promise is `truthy`. But we should add the await and update to `t.true`. Great catch